t/api/04-jobs.t

```diff
@@ -23,7 +23,7 @@ use File::Temp;
 use Test::Mojo;
 use Test::Output;
 use Test::Warnings ':report_warnings';
-use OpenQA::Test::TimeLimit '180';
+use OpenQA::Test::TimeLimit '20';
```

Comment:
This seems to be too little (as seen on this PR's CI results)

Reply:
this multiplies to 120s. I don't see a problem with current CI results. However, you might have found an earlier failed job when I still had 10s timeout